FAQ.md

```diff
@@ -30,40 +30,53 @@ We like getting inspired on [codepen.io](https://codepen.io) and building protot
 
 ## Technical FAQ
 
-### I don't know how to test my module within the COBI.bike app
+### I don't know how to test my module within the COBI.Bike app
 
 Did you already sign up as a developer on [my.cobi.bike](https://my.cobi.bike)? If yes, please follow the instructions on the [DevKit site](https://github.com/cobi-bike/COBI-DevKit).
 
 ### Sometimes the settings of my module are reset
 
-If your iOS device is low on storage, the Safari browser cache may get deleted. This is a known issue for now. 
+If your iOS device is low on storage, the Safari browser cache may get deleted. This is a known issue for now.
 
 ### I'm using the `subscribe` method, but I don't receive any data
 
 Make sure to authenticate your module first – to do so call
 `COBI.init("YOUR-TOKEN")` before you subscribe to any `COBI.js` data stream.
 
-### Any tips for debugging my module within the COBI.bike app on the phone?
+### I changed the code of the module but the changes are not appearing on the COBI.Bike app
+
+We have noticed that sometimes there are some browser-related caching issues so this is what might be happening to you. The easiest solution would be to go into the module selection screen, remove *My Module* and add it again.
+
+### Any tips for debugging my module within the COBI.Bike app on the phone?
 
 The native app will display javascript errors in a dialog. Just activate this feature in the diagnostics settings of the app.
 
 ### How do I get a specific version of the `COBI.js` library?
 
-The following url provides `0.42.0` of `COBI.js`: *https://cdn.cobi.bike/cobi.js/0.42.0/cobi.js*. 
+The following url provides `0.42.0` of `COBI.js`: *https://cdn.cobi.bike/cobi.js/0.42.0/cobi.js*.
 This will work for any other published version.
 
+### Are there any other known technical limitations?
+
+There are not many technical constraints to module development; any modern framework can be used (Angular, React, Vue) as well as any mobile-compatible library.
+
+There are however some limitations that need to be taken into account:
+
+* The webapp should not rely on server side rendering.
```

Comment:
where do these restrictions come from ? I guess this is oriented towards offline support but AFAIK these are not limitations but design decisions of the module that we want to push right ?

Reply:
yeah but my point is that server-side is not allowed because "we want to support offline mode right?" but on the next item says that "the user needs to be online" which contradicts the previous restriction, which is why I am questioning these restrictions